docs/sql_cheatsheet.md

```diff
@@ -114,7 +114,7 @@ SET is_open=0,
   close_date='2017-12-20 03:08:45.103418',
   close_rate=0.19638016,
   close_profit=0.0496,
-  close_profit_abs = (amount * 0.19638016 * (1 - fee_close) - (amount * open_rate * 1 - fee_open)
```

Comment:
would you mind fixing this in line 104 too? it's the "abstract" sample, but it's just as wrong ...

Reply:
@xmatthias Of course not! I fixed that one as well.